hw/ip/rom_ctrl/data/rom_ctrl_testplan.hjson

```diff
@@ -63,6 +63,29 @@
       milestone: V3
       tests: []
     }
+    {
+      name: unsuccessful_rom_chk
+      desc: '''
+            Corrupt the expected digest so that there is a mismatch between expected digest and 
+            KMAC digest. 
```

Comment:
Sound like a good idea, although I'd be equally happy to fold this into a generic test (if there were one) and track the mismatch with a functional coverage point. The laziest approach is probably to put this in the existing smoke test with 50% probability :-)

Possibly "corrupt" is the wrong term here: we're just going to be generating a ROM image with an expected checksum that doesn't match. "Corrupt" sounds a bit like we're injecting a fault during the run. How about "Choose an expected digest so that..." ?

Reply:
After our discussion, I separated the successful rom check from the smoke test.